front-end/stateofthecards-front-end/src/Login.tsx

```diff
@@ -0,0 +1,99 @@
+import React, { Component } from "react";
+import styles from "./Login.module.css";
+import { Redirect } from "react-router-dom";
+
+interface IProps {}
+
+interface IState {
+	post: boolean;
+	username: string;
+	password: string;
+}
+
+class Login extends Component<IProps, IState> {
+	constructor(props: IProps) {
+		super(props);
+		this.state = { post: false, username: "", password: "" };
+	}
+
+	handleLogin = (event: any) => {
+		event.preventDefault();
+
+		// Instead of printing it to the console
+		//  Login to the back-end
+		console.log(this.state.username, this.state.password);
+
+		// When succesfull, re-direct the user
+		//  To the dashboard.
+		this.setState({ post: true });
+	};
+
+	handleUsernameInput = (event: any) => {
+		this.setState({
+			username: event.target.value,
+		});
+	};
+
+	handlePasswordInput = (event: any) => {
```

Comment:
The ‘handleXInput’ eventhandlers should be named onXChange following convention. It’s also more logical since you should fire them on every change